noaa/challenges/default.json

```
@@ -107,7 +107,9 @@
           "warmup-iterations": 100,
           "iterations": 500,
           "target-throughput": 5
-        }
+        }{%- if include_aggs is defined and include_aggs | default(false) %},
```

this can be simplified, i think
```suggestion
        }{%- if include_aggs | default(false) %},
```